### ISSUE TITLE
Update doc about outbound calls

### ISSUE THREAD

**Issue by JimDaly** [MicrosoftDocs/powerapps-docs#969]
Outbound parallel calls are fine. Callbacks to CDS shouldn't be done in parallel.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0c72d23e-97ca-b6c7-6415-e4bb12e2e022
* Version Independent ID: 3ae1f0ac-0937-94f0-ebbd-c502b0db744e
* Content: [Do not use parallel execution within plug-ins and workflow activities - PowerApps](https://docs.microsoft.com/en-us/powerapps/developer/common-data-service/best-practices/business-logic/do-not-use-parallel-execution-in-plug-ins#feedback)
* Content Source: [powerapps-docs/developer/common-data-service/best-practices/business-logic/do-not-use-parallel-execution-in-plug-ins.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/live/powerapps-docs/developer/common-data-service/best-practices/business-logic/do-not-use-parallel-execution-in-plug-ins.md)
* Service: **powerapps**
* GitHub Login: @JimDaly
* Microsoft Alias: **pehecke**

**Comment by phecke** [comment]
@JimDaly 

Thanks for your feedback. We are closing this issue because we are either unable to find relevant information or reproduce this. If you're still experiencing the issue described here, please reopen or create a new issue.
